packages/vsg/package.yaml

```
@@ -0,0 +1,16 @@
+---
+name: vsg
+description: VHDL Style Guide (VSG), Coding style enforcement for VHDL.
+homepage: https://pypi.org/project/vsg/
+licenses:
+  - GPL-3.0
+languages:
+  - VHDL
+categories:
+  - Formatter
```

Technically seems like a linter also, might as well add it :)
```suggestion
  - Formatter
  - Linter
```